javac-plugin/src/test/java/org/moditect/deptective/plugintest/basic/baranparam/BarAnnotationParameter.java

```diff
@@ -0,0 +1,5 @@
+package org.moditect.deptective.plugintest.basic.baranparam;
```

Comment:
Can you add the license header (`mvn license:format`). I'm surprised that the build didn't fail due to the missing header?